src/config.rs

```diff
@@ -124,6 +124,9 @@ pub struct ConfigBuilder {
 
     /// Close the connection if no substreams are open within this time frame.
     keep_alive_timeout: Duration,
+
+    /// True if litep2p should attempt to dial local addresses.
+    use_private_ip: bool,
```

Comment:
```suggestion
    dial_global_ips_only: bool,
```
?

private_ip != !global_ip